dhctl/cmd/dhctl/commands/converge.go

```diff
@@ -29,6 +31,31 @@ import (
 	"github.com/deckhouse/deckhouse/dhctl/pkg/terraform"
 )
 
+func getCacheIdentityFromKubeconfig(
+	kubeconfigPath string,
+	kubeconfigContext string,
+) (string, error) {
+	builder := strings.Builder{}
+	builder.WriteString("kubeconfig")
+
+	kubeconfigName := filepath.Base(kubeconfigPath)
+	if kubeconfigName == "." || kubeconfigName == ".." || kubeconfigName == "/" {
```

Comment:
What if it has a quote sign or a double slash? Why did you pick these three options?

Reply:
Same question